package.nls.json

```diff
@@ -110,6 +110,9 @@
     "AWS.command.cdk.help": "View CDK Documentation",
     "AWS.command.ec2.openTerminal": "Open terminal to EC2 instance...",
     "AWS.command.ec2.openRemoteConnection": "Connect to EC2 instance in New Window...",
+    "AWS.command.ec2.startInstance": "Start EC2 Instance...",
+    "AWS.command.ec2.stopInstance": "Stop EC2 Instance...",
+    "AWS.command.ec2.rebootInstance": "Reboot EC2 Instance...",
```

Comment:
"..." indicates that more steps are involved, is that the case for all of these? (A confirm dialog wouldn't count as "more steps".)

Reply:
They don't have more steps, so I went ahead and removed them.